WTWSMS/common/government_flavor/WtWSMS_government_flavor.txt

```diff
@@ -34,4 +37,26 @@ government_flavor = {
 	trigger = {
 		government = legion_government
 	}
-}
\ No newline at end of file
+}
+government_flavor = {
+	name = gov_bureaucracy
+	graphics = GFX_government_flavor_bureaucracy_placeholder
```

Comment:
There is also a big difference between the art we would like to display and the art we have. I guess I could recycle a loading-screen.